src/Compiler/Microsoft.AspNetCore.Razor.Language/src/Legacy/ParserHelpers.cs

```diff
@@ -15,25 +12,23 @@ internal static class ParserHelpers
 {
     public static bool IsNewLine(char value)
```

Comment:
Are these methods explicitly about C# only? If so, we should probably track removing them with https://github.com/dotnet/razor/issues/8445 as well.

Reply:
I'm not sure. These are in the Legacy folder, so I don't know if they're used broadly.

Reply:
Gotcha. Why don't we just leave a comment in the issue to rationalize their use.

Reply:
Agreed. I added a note. Thanks!